### ISSUE TITLE
pulseaudio 15.0

### ISSUE THREAD

**Issue by yu-re-ka** [NixOS/nixpkgs#142546]

**Comment by kolaente** [comment]
I've tried to upgrade it but hit a wall, I keep getting this error while building:

```
meson.build:676:0: ERROR: C shared or static library 'wrap' not found
```

My efforts so far are here: https://github.com/kolaente/nixpkgs/blob/update/pulseaudio-15.0/pkgs/servers/pulseaudio/default.nix

**Comment by kolaente** [comment]
Looks like pulseaudio switched their build system from make to meson + ninja

**Comment by kero0** [comment]
I've hacked this together based off of yours. I'm fairly sure this won't It's in no way clean but as far as I can tell it mostly works for me?? I noticed some popping and I'm not sure of that's a config issue or just a bug with pulseaudio 15.0. 
Somebody else who's better at packaging for nix and cares about making it clean can use this as a launch point if they want. 

https://gist.github.com/kero0/13cf1b1af40b5f7e4df29ad35a7cd9bd
